README.rst

```diff
@@ -17,7 +17,3 @@ Kinto
 
 Kinto is a server allowing you to store and synchronize arbitrary data,
 attached to your Firefox account.
-
-* `Online documentation <http://kinto.readthedocs.org/en/latest/>`_
```

Comment:
yeah, and this probably is the reason why we have these links here. Maybe it's time to split the README and the actual doc.